run_migrations.py

```
@@ -0,0 +1,50 @@
+MIGRATIONS_DIR = "migrations"
+
+logger = logging.getLogger(__name__)
+
+
+async def ensure_tortoise_connected():
+    await Tortoise.init(config=TORTOISE_ORM)
+    await Tortoise.generate_schemas(safe=True)
+    await Tortoise.close_connections()
+
+
+def run_command(cmd):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)